src/htmlEntities.ts

```diff
@@ -0,0 +1,6 @@
+const entities: { [key: string]: string } = {
+  '<': '&lt;',
+  '>': '&gt;',
+};
+
+export const encodeHtml = (str: string) => str.replace(/[<>]/g, tag => entities[tag] || tag);
```

Comment:
We have already `marked` as dependency maybe we can rely on it to escape the HTML.

Reply:
i don't think marked has html encoding? https://github.com/markedjs/marked/issues/1232